methods/index.js

```diff
@@ -53,28 +53,413 @@ export const ethGetBalance = {
   params: ["0x6743938A48fC8799A5608EF079C53f3cF3B84398", "latest"],
 };
 
+export const ethGetBlockByNumber = {
+  name: "eth_getBlockByNumber",
+  params: ["latest", false],
+};
+
+export const ethBlockNumber = {
+  name: "eth_blockNumber",
+  params: [],
+};
+
+export const ethGetLogs = {
+  name: "eth_getLogs",
+  params: [{ fromBlock: "latest", address: "0x0000000000000000000000000000000000000000" }],
+};
+
+export const ethGetTransactionReceipt = {
+  name: "eth_getTransactionReceipt",
+  params: [
+    "0xb0ccffb14e1e79f0b6c6ceec223bb4bbf6fc302ae76067f1f9225bae9d6f3fdc",
```

Comment:
I'd also like to mention it here so it doesn't get lost. It's okay for specific parameters to be hard-coded (though they still should be configurable without touching this file).

1. It should be possible to run benchmarks on both calibnet and mainnet. 
2. Tests should not break over time. This transaction will likely be unavailable on nodes bootstrapped with a minimal snapshot. This is a blocker for any real usage of this benchmark.

The limitations should be clearly stated in the README, not to give false impressions and allow potential users to resolve their issues more easily.

For now, let's at least group some of the parameters, especially those that appear a few times, into named constants.